### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**
Some of them might be breaking changes, so it might need to be broken up if there are particular parts seem iffy on review

### CODE REVIEW

File: config/entityProviders.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "organizationsettings": "env://ENTITY_PROVIDER_ORGANIZATION_SETTINGS?default=firstconfigured",
   "organizationmembercache": "env://ENTITY_PROVIDER_ORGANIZATION_MEMBER_CACHE?default=firstconfigured",
   "repository": "env://ENTITY_PROVIDER_REPOSITORY?default=firstconfigured",
-  "repositorycache": "env://ENTITY_PROVIDER_REPOSITORY_CACE?default=firstconfigured",
+  "repositorycache": "env://ENTITY_PROVIDER_REPOSITORY_CACHE?default=firstconfigured",
```
Comment (nschonni) [inline config/entityProviders.json:9]: Maybe a deployment issue to create a new cache

File: routes/org/repos.ts [hunk 2]
```diff
@@ -779,7 +779,7 @@ async function triggerRenameNotification(
   output: ITemporaryCommandOutput[]
 ): Promise<void> {
   const { config, insights, operations, mailAddressProvider, viewServices } = providers;
-  insights.trackMetric({ name: 'RenameDefaultBranchs', value: 1 });
+  insights.trackMetric({ name: 'RenameDefaultBranches', value: 1 });
```
Comment (nschonni) [inline routes/org/repos.ts:782]: Not sure if this will skew the existing metrics

Comment (jeffwilcox) [reply routes/org/repos.ts:782]: Nice! This shipped way before GitHub implemented native branch renaming, and at least on the Microsoft side, we don't encourage use of this feature anymore, so it's fine from my side!

On the basis of removing complexity, might make sense to cut the rename default branch code entirely.

Opened https://github.com/microsoft/opensource-management-portal/issues/415

Comment (nschonni) [reply routes/org/repos.ts:782]: No worries, if you want to trim out that code, I can rebase this afterwards.

File: config/entityProviders.json (hunk 1 above)
Comment (jeffwilcox) [reply config/entityProviders.json:9]: Oh that's embarrassing! Yeah, a deployment issue if anyone has over-ridden that entity.

Might be time for a `changelog.md` to help track version changes of note and at least increment the minor version of the package or something like that...

**Comment by nschonni** [comment]
BTW, I used cSpell for this and could wire it into the scripts and CI if you want

File: routes/org/repos.ts (hunk 2 above)
Comment (jeffwilcox) [reply routes/org/repos.ts:782]: I'll take this as-is for now, thanks!

**Review by jeffwilcox — APPROVED** [review]

**Comment by jeffwilcox** [comment]
> 

Thanks! Yeah I love automating those things over time, similar to Markdown authoring pack in vscode etc.

Nick, would you be interested in me granting you "read" access as an outside collaborator on this repo? With that, you can also use Codespaces at our cost, if that'd be interesting.

**Comment by nschonni** [comment]
Wouldn't hurt if you want to tag me on anything, but I'd probably stick to my local setup, since I've got all my plugins/tools already 😉

**Pull request merged**